node/network/protocol/src/request_response/mod.rs

```diff
@@ -106,6 +123,8 @@ impl Protocol {
 			// assuming we can service requests relatively quickly, which would need to be measured
 			// as well.
 			Protocol::AvailabilityFetching => 100,
+			// 10 seems reasonable, considering group sizes of max 10 validators.
+			Protocol::CollationFetching => 10,
```

Comment:
what happens when the channel is full? do we drop messages or stall with .await?

Reply:
Messages would get dropped, but this would only happen if a collator is really overwhelmed with handling requests. Even if eleven validators would request at the same time, this would likely not happen if the collator acts reasonably quickly.

See [here](https://github.com/paritytech/substrate/blob/18ab0903c23eabbe53860b810314e8d4102091d8/client/network/src/request_responses.rs#L105) for details.